pkg/operators/oci-handler/extrainfo.go

```diff
@@ -0,0 +1,103 @@
+package ocihandler
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/distribution/reference"
+	ocispec "github.com/opencontainers/image-spec/specs-go/v1"
+	"sigs.k8s.io/kustomize/kyaml/yaml"
+
+	"github.com/inspektor-gadget/inspektor-gadget/pkg/gadget-service/api"
+	"github.com/inspektor-gadget/inspektor-gadget/pkg/operators"
+)
+
+type Config struct {
+	Name             string                      `yaml:"name"`
+	Description      string                      `yaml:"description"`
+	HomepageURL      string                      `yaml:"homepageURL"`
+	DocumentationURL string                      `yaml:"documentationURL"`
+	SourceURL        string                      `yaml:"sourceURL"`
+	Datasources      map[string]Datasource       `yaml:"datasources"`
+	Params           map[string]map[string]Param `yaml:"params"`
+}
+
+type Datasource struct {
+	Fields map[string]Field `yaml:"fields"`
+}
+
+type Field struct {
+	Annotations map[string]interface{} `yaml:"annotations"`
+}
+
+type Param struct {
+	Key          string `yaml:"key"`
+	DefaultValue string `yaml:"defaultValue"`
+	Description  string `yaml:"description"`
+}
+
+func addExtraInfo(gadgetCtx operators.GadgetContext, metadata []byte, manifest *ocispec.Manifest) {
+	parsed, err := reference.Parse(gadgetCtx.ImageName())
+	if err != nil {
+		fmt.Println("Error parsing image name")
```

Comment:
Errors should not be printed on stdout. Either send them in extraInfo or skip them. Ditto line 69.